store/sqlstore/supplier.go

```diff
@@ -28,10 +29,12 @@ import (
 )
 
 const (
-	INDEX_TYPE_FULL_TEXT = "full_text"
-	INDEX_TYPE_DEFAULT   = "default"
-	DB_PING_ATTEMPTS     = 18
-	DB_PING_TIMEOUT_SECS = 10
+	INDEX_TYPE_FULL_TEXT       = "full_text"
+	INDEX_TYPE_DEFAULT         = "default"
+	PG_DUP_TABLE_ERROR_CODE    = "42P07"      // see https://github.com/lib/pq/blob/master/error.go#L268
+	MYSQL_DUP_TABLE_ERROR_CODE = uint16(1050) // see https://fromdual.com/mysql-error-codes-and-messages-1050-1099#error_er_table_exists_error
```

Comment:
I would prefer referring to the official MySQL reference- https://dev.mysql.com/doc/mysql-errors/5.7/en/server-error-reference.html#error_er_table_exists_error

Reply:
I do not know why i pasted that link there :D. Thanks, fo catching this up 🦅 👀